Sources/MobileCoinClient.swift

```diff
@@ -217,11 +218,73 @@ public final class MobileCoinClient {
         }
     }
 
+    public func prepareTransaction(
```

Comment:
I lean towards not adding these extra functions to the public API, we could modify the existing API and provide a default value in the function signature. Anyone currently calling the api won't have to change anything.

Reply:
Sounds good - I was kind of thinking the same...

Reply:
consolidated rng & non-rng APIs via optional param defaulting to nil, where implementation will use the internal MobileCoinDefaultRng().